src/main/java/net/pms/database/TableFilesStatus.java

```diff
@@ -203,7 +204,7 @@ public static FilesStatusResult isFullyPlayed(final String fullPathToFile) {
 		FilesStatusResult result;
 
 		try (Connection connection = database.getConnection()) {
-			String query = "SELECT ISFULLYPLAYED FROM FILES JOIN " + TABLE_NAME + " ON FILES.ID = " + TABLE_NAME + ".FILEID WHERE FILENAME = " + sqlQuote(sqlLikeEscape(fullPathToFile));
+			String query = "SELECT ISFULLYPLAYED FROM FILES JOIN " + TABLE_NAME + " ON FILES.ID = " + TABLE_NAME + ".FILEID WHERE FILENAME = " + sqlQuote(fullPathToFile);
 
```

Comment:
"LikeEscape" should only be used with ```LIKE```, not when using ```=```. The escaped syntax isn't necessarily valid when using ```=```.